server/lobbyconnection.py

```diff
@@ -628,12 +556,12 @@ def check_version(self, message):
                                      "player_id": player_id
                                  })
 
-            if not self.player_service.is_uniqueid_exempt(player_id):
-                # UniqueID check was rejected (too many accounts or tamper-evident madness)
-                uniqueid_pass = await self.validate_unique_id(cursor, player_id, steamid, message['unique_id'])
-                if not uniqueid_pass:
+            if not self.player_service.is_uniqueid_exempt(player_id) or steamid is not None:
```

Comment:
Check conformity if steamid is not None?
I'd expect `and steamid is None`

Reply:
fixed